index.js

```diff
@@ -97,6 +97,11 @@ class PQueue {
 			}
 		});
 	}
+
+	clear() {
+		this.queue._queue = [];
```

Comment:
i think i just just set it to a new queue because if the custom queue has a side effect i dont want to trigger that